mu-plugins/blocks/query-filter/src/view.js

```diff
@@ -58,6 +84,7 @@ wpStore( {
 					} else {
 						context.wporg.queryFilter.isOpen = true;
 						document.documentElement.classList.add( 'is-query-filter-open' );
+						toggleOverflowX();
 					}
```

Comment:
It seems like we can just call this, rather than doing it for each of open and close.

```suggestion
					}
					
					toggleOverflowX();
```